### PR TITLE
Review and update portfolio walkthrough

### PR DESCRIPTION
**Pull request by jaredjj3**
The students will come from a variety of skill levels. I thought it would be helpful to add some context on what `ctrl + c` does.

I also wanted to encourage students to get in the good habit of committing early-and-often. I added some git hygiene tips when they're at the point of making changes themselves.

### CODE REVIEW

**Comment by jaredjj3** [comment]
@googlebot I signed it!

File: walkthroughs/week-1-web-development/portfolio-walkthrough.md [hunk 1]
```diff
@@ -104,6 +104,9 @@ to include your name.
 
 ### Restarting Your Development Server
 
+**Tip:** Pressing `ctrl + c` in the console will interrupt any process running
+in the foreground, which willusually cause the program to abort.t
```
Comment (KevinWorkman) [inline walkthroughs/week-1-web-development/portfolio-walkthrough.md]: nit: Can you add a space?

Comment (KevinWorkman) [inline walkthroughs/week-1-web-development/portfolio-walkthrough.md]: Is this extra "t" a typo?

File: walkthroughs/week-1-web-development/portfolio-walkthrough.md [hunk 2]
```diff
@@ -286,6 +289,45 @@ When you see `Dev App Server is now running` in the console, click the
...
+
+Replace FILEPATH with the path/to/the/changed/file. To stage a file, run:
+
+```
+git add FILEPATH
+```
+
+You can stage multiple files at once.
+
+To check the sttaged files, run:
```
Comment (KevinWorkman) [inline walkthroughs/week-1-web-development/portfolio-walkthrough.md]: Typo?

File: walkthroughs/week-1-web-development/portfolio-walkthrough.md (hunk 1 above)
Comment (jaredjj3) [reply walkthroughs/week-1-web-development/portfolio-walkthrough.md]: Done.

Comment (jaredjj3) [reply walkthroughs/week-1-web-development/portfolio-walkthrough.md]: Yes, done.

File: walkthroughs/week-1-web-development/portfolio-walkthrough.md (hunk 2 above)
Comment (jaredjj3) [reply walkthroughs/week-1-web-development/portfolio-walkthrough.md]: Yes, done.

**Comment by jaredjj3** [comment]
Thanks for catching those mistakes, Kevin. It looks like I need to get my keyboard fixed. :)

**Pull request merged**